src/test/jtx/Env_test.cpp

```diff
@@ -748,8 +748,12 @@ class Env_test : public beast::unit_test::suite
             params[jss::fee_mult_max] = 1;
             params[jss::fee_div_max] = 2;
             // RPC errors result in telENV_RPC_FAILED
```

Comment:
I feel this comment is more appropriate in `src/test/jtx/rpc.h`. Specifically, in the `operator()` of the `rpc` class.

Reply:
Fixed in f58d42de8b